lang/r/projects.txt

```diff
@@ -0,0 +1,30 @@
+https://github.com/binder-examples/r
```

Comment:
I would have expected dlpyr and ggplot2 to be in this list (or those R packages are not in github maybe?)